testsuite/MDAnalysisTests/lib/test_cutil.py

```diff
@@ -46,6 +45,43 @@ def test_unique_int_1d(values):
     assert res.dtype == ref.dtype
 
 
+@pytest.mark.parametrize(['array', 'unique_array', 'inverse'],  (
+    ([], [], []),  # empty array
+    ([1, 1, 1, 1, ], [1,], [0, 0, 0, 0, ]),  # all identical
+    ([2, 3, 5, 7, ], [2, 3, 5, 7, ], [0, 1, 2, 3, ]),  # unique, sorted
+    ([5, 2, 7, 3, ], [5, 2, 7, 3, ], [0, 1, 2, 3, ],),  # unique, unsorted
+    ([1, 2, 2, 4, 4, 6, ], [1, 2, 4, 6, ], [0, 1, 1, 2, 2, 3, ]),  # duplicates, sorted
```

Comment:
```suggestion
    # duplicates, sorted
    ([1, 2, 2, 4, 4, 6], [1, 2, 4, 6], [0, 1, 1, 2, 2, 3]),
```

The PEP8 bot is complaining about some line lengths here -- one option could be to move the comments (like the `# duplicates, sorted`) ones to the lines above